teamscale-gradle-plugin/MIGRATION_35.md

```diff
@@ -0,0 +1,80 @@
+# Migration guide for the `com.teamscale` Gradle plugin version 34 to 35
+Previously, the plugin provided a way to specify the partitions either globally for a project or on a per task level.
+Additionally, a `teamscaleReportUpload` was provided automatically on the root project that would lazily pick up all the reports and upload them to Teamscale.
+
+Due to the restrictions that are imposed by the Gradle configuration cache, we had to adjust the API of the plugin to provide compatibility with it.
+
+
+## Usage of `Property` for all inputs
+We switched all extension and task properties from plain types e.g. `boolean` to `Property<Boolean>` etc.
+In newer Gradle versions this should be handled transparently.
+If you are still using an older version, you might need to replace `partition = "Something"` with `partition.set("Something")`.
```

Comment:
do you know how old? otherwise it remains mysterious to the reader if they are affected or not ;-)

Reply:
Good point! I was in a hurry when I wrote this so didn't take the time to look up the details. Actually Groovy always supported this and Gradle 8.2 also enabled this by default for Kolin DSL. As we only support Gradle 8.4+ from now on, I just dropped that part.